doc/source/data/key-concepts.rst

```diff
@@ -153,10 +153,14 @@ tasks launched from within a placement group will be executed outside of that pl
 group by default. This can be thought of as Datasets requesting resources from the
 margins of the cluster, outside of those ML library placement groups.
 
-Although this is the default behavior, you can force all Datasets workloads to be
-scheduled within a placement group by specifying a placement group as the global
-scheduling strategy for all Datasets tasks/actors, using the global
-:class:`DatasetContext <ray.data.DatasetContext>`.
+To avoid hangs or CPU starvation of Datasets when used with Tune or Train, you can
+exclude a fraction of CPUs from placement group scheduling, using the
+``_max_cpu_fraction_per_node`` placement group option (Experimental).
+
+.. warning::
+
+    ``_max_cpu_fraction_per_node`` is experimental and not recommended for use with
+    autoscaling clusters.
```

Comment:
Should we be more explicit and say that the reason is that doing so may cause deadlock?